Aztec/Classes/TextKit/TextView.swift

```diff
@@ -724,6 +767,56 @@ open class TextView: UITextView {
     }
 
 
+    /// Removes the List Attributes from a collection of attributes, whenever:
+    ///
+    ///     A. The caret is at the very end of the document (+ there is no selected text)
```

Comment:
If the selected range location starts at EOF, then `(+ there is no selected text)` is always true.